index.html

```diff
@@ -1,6 +1,22 @@
-﻿<!doctype html>
+<!DOCTYPE html>
 <html>
 <head>
+    <title>BTV: Who's My Councillor?</title>
+    
+    /* Standard search engine metas */
+    <meta charset="UTF-8">
+    <meta name="description" content="Enter your address to find your City Ward & District councillors">
+    <meta name="keywords" content="ward, wards, district, districts, council, councillor, councillors, burlington, vt, church, street, vermont, btv, queen, city">
+    <meta name="author" content="@rfreeburn">

```

Comment:
I just made a new commit, but I'm not sure if it's in the group of the pull request...